### PR TITLE
refactor(mesh): social page infinite scroll and fix bugs

### PR DESCRIPTION
**Pull request by tyc67**
## Notable Changes

- 社群頁接 pagination api [doc](https://paper.dropbox.com/doc/MongoDB--CY01JcYayXAJ2ZknVQFW_VrSAg-4yv9QHuFJqqn12dag3TUN)
- readr mesh logo 更新
- 新增 Aavatar fallback
- 調整登入後 redirect 機制
- 調整社群頁 `<Feed/>` displayPicks 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208463265770701
  - https://app.asana.com/0/0/1208535128966412
  - https://app.asana.com/0/0/1208558093061128
  - https://app.asana.com/0/0/1208516555415399
  - https://app.asana.com/0/0/1208488184161594

### CODE REVIEW

File: packages/mesh/app/login/_components/login-steps.tsx [hunk 1]
```diff
@@ -42,8 +42,9 @@ export default function LoginSteps() {
         if (response.result === 'sign-up') {
           setStep('set-name')
         } else if (response.result === 'logged-in') {
+          const redirectRoute = localStorage.getItem('login-redirect') ?? '/'
```
Comment (caesarWHLee) [inline packages/mesh/app/login/_components/login-steps.tsx:45]: 用完 `login-redirect` 清掉該值，避免未處理情況登入後跳轉到錯誤頁面

File: packages/mesh/components/story-card/avatar.tsx [hunk 2]
```diff
@@ -33,16 +36,18 @@ export default function Avatar({
...
     <Image
       className={`${avatarClass} inline-block ${
         isRound && 'rounded-full'
       } bg-white ring-2 ring-white ${extra} object-cover`}
-      src={src}
+      src={imgSrc}
       width={sideLength}
       height={sideLength}
       alt={src}
+      onError={() => setImgSrc('/images/default-avatar-image.png')}
```
Comment (caesarWHLee) [inline packages/mesh/components/story-card/avatar.tsx:51]: 這部分處理建議改用河馬 [PR](https://github.com/readr-media/Sachiel/pull/883/files) 中 `image-with-fallback` 的做法，看要不要加個 TODO 等那個 PR merge 後再統一修改

File: packages/mesh/app/login/_components/login-steps.tsx (hunk 1 above)
Comment (tyc67) [reply packages/mesh/app/login/_components/login-steps.tsx:45]: 修改於 [a182973](https://github.com/readr-media/Sachiel/pull/882/commits/a182973920a08fde7f4c8b55cb43acee1256f8aa)

File: packages/mesh/components/story-card/avatar.tsx (hunk 2 above)
Comment (tyc67) [reply packages/mesh/components/story-card/avatar.tsx:51]: 修改於 [a182973](https://github.com/readr-media/Sachiel/pull/882/commits/a182973920a08fde7f4c8b55cb43acee1256f8aa)

**Review by caesarWHLee — APPROVED** [review]

**Review by nihclil — APPROVED** [review]

**Pull request merged**